src/core/analysis-context.ts

```diff
@@ -224,16 +225,16 @@ export class AnalysisContext {
     }
     const scannedDocument = this._cache.scannedDocuments.get(resolvedUrl);
     if (!scannedDocument) {
-      return <Warning>{
+      return {
         sourceRange: {
-          file: this.resolveUrl(url),
+          file: resolvedUrl,
```

Comment:
Is the Warning interface not capable of matching this because the branded string is given as the `file` property here in the `sourceRange` or was there a different reason for using `as any` instead of `as Warning`?

Reply:
Good eye, this code is actually just wrong, but I split the fix out into https://github.com/Polymer/polymer-analyzer/pull/741